recipe/build.sh

```diff
@@ -63,11 +70,18 @@ cmake -LAH                                                                \
     -DCMAKE_PREFIX_PATH=${PREFIX}                                         \
     -DCMAKE_INSTALL_PREFIX=${PREFIX}                                      \
     -DCMAKE_INSTALL_LIBDIR="lib"                                          \
+    -DCMAKE_AR="${AR}"                                                    \
+    -DCMAKE_LINKER="${LD}"                                                \
```

Comment:
I think this is not defined in the legacy compiler case. Hence the failure there. That may be true of a few of these as well (e.g. `AR`, `NM`, etc.).

Reply:
Adding PR ( https://github.com/conda-forge/toolchain-feedstock/pull/53 ) to handle setting these for the legacy toolchain.